Reactor/Networking/HandshakePopup.cs

```diff
@@ -0,0 +1,16 @@
+namespace Reactor.Networking;
+
+internal static class HandshakePopup
+{
+    public const string Message =
+        """
+        This server doesn't support modded handshake.
```

Comment:
```suggestion
        This server doesn't support Reactor's modded handshake.
```

This leads to the word "handshake" not being on it's own line, a so-called [orphan](https://en.wikipedia.org/wiki/Widows_and_orphans). By adding another word that issue is fixed:

![2023-12-08T19:24:21,597363820+01:00](https://github.com/NuclearPowered/Reactor/assets/5243971/87844e78-688f-4226-9562-422776474357)